modules/web/js/ballerina/diagram2/views/default/components/decorators/canvas-decorator.jsx

```diff
@@ -55,6 +56,18 @@ class CanvasDecorator extends React.Component {
                     {/* This space is used to render html elements over svg*/ }
                 </div>
                 {(this.props.annotations && this.props.annotations.length > 0) ? this.props.annotations : null }
+                <CSSTransitionGroup
```

Comment:
we should apply transitions for individual components. lets refactor after merge.